vaadin-checkbox-flow-parent/vaadin-checkbox-flow-integration-tests/src/test/java/com/vaadin/flow/component/checkbox/tests/CheckboxGroupIT.java

```diff
@@ -191,9 +191,6 @@ public void groupHasLabelAndErrorMessage_setInvalidShowEM_setValueRemoveEM() {
                 "Correct error message should be shown after the button clicks",
                 "Field has been set to invalid from server side",
                 errorMessage.getText());
-
-        executeScript("arguments[0].value=['2'];", group);
-        verifyGroupValid(group, errorMessage);
```

Comment:
This test had the wrong assumption that setting a value on the client should make the component valid. That's not correct, as long as the component has been set to invalid, it should stay invalid.